### DISCUSSION TITLE
nonebot2如何添加bot实例

### DISCUSSION THREAD

**Discussion by githablocal** [nonebot/discussions#373] [问答] [answered]
TOKEN或用户名密码配置之类的，目前的文档里应该没有提？
另外有无快速创建bot.py的方法或模板文件，除了从文档里拷贝？

**Comment by yanyongyu** [comment, marked as answer]
？nonebot本体又不管这个，你要配用户名密码那是适配器的事，请看对应适配器的文档。

`bot.py` 不推荐创建，除非你有明确的需求，你可以用`nb generate`生成

**Reply by githablocal** [reply]
了解了，谢谢！Telegram适配器是内置还是在某个项目呢？README的直接链接到tg的http api文档了

**Reply by yanyongyu** [reply]
文档以及商店里都有说明和链接。。

**Reply by githablocal** [reply]
找到了，原来就在组织的repo列表下
